### PR TITLE
refactor: modernize python script to latest GCP APIs

### PR DESCRIPTION
**Pull request by peter0083**
docs: update readme to reflect lastest API change and reference to DG

This issue addresses issue #1 

changes in this PR:
- updated codes imported from google cloud monitoring, labels, time series and metrics.

### CODE REVIEW

File: monitor.py [hunk 1]
```diff
@@ -1,285 +1,358 @@
...
-from google.cloud.monitoring_v3.types import LabelDescriptor, MetricDescriptor, TimeSeries
+from google.cloud.monitoring_v3 import TimeSeries, TimeInterval, Point
 import json
 from os import environ
 import psutil as ps
 import requests
 from signal import signal, SIGTERM
-from sys import stderr
 from time import sleep, time
+import json
```
Comment (ovesh) [inline monitor.py:15]: Are you following the wiki [style guide](https://deepgenomics.atlassian.net/wiki/spaces/DG/pages/591626241/DG+Python+Style+Guide)? In particular the webhook setup. (In particular, I would suggest also running isort in your IDE).

File: monitor.py [hunk 2]
```diff
@@ -1,285 +1,358 @@
...
+        price = pricelist[price_key][machine["region"]] * disk["sizeGb"]
+        if disk["type"].startswith("pd"):
+            price /= 730  # hours per month
+        total += price
+    return total
+
 
 def reset():
-  global memory_used, disk_used, disk_reads, disk_writes, last_time
+    global memory_used, disk_used, disk_reads, disk_writes, last_time
```
Comment (ovesh) [inline monitor.py:111]: Not related to this change, but let's create an issue to get rid of `global`.

File: monitor.py [hunk 3]
```diff
@@ -1,285 +1,358 @@
...
+    else:
+        return {
+            "type": "local-ssd",
+            "sizeGb": 375,
+        }
+
+
+PRICELIST_JSON = (
+    "pricelist.json"  # json that contains GCP pricing. used for cost metric.
+)
```
Comment (ovesh) [inline monitor.py:63]: Did `black` add the parentheses? I wonder if it would still do that if you added the comment _above_ the line

**Review by ovesh — APPROVED** [review]

**Comment by ovesh** [comment]
I hope this repo has good unit tests.

**Comment by peter0083** [comment]
> I hope this repo has good unit tests.

This repo does not have unit tests. I have opened an issue to flag this problem #4 .

**Review by ovesh — APPROVED** [review]

**Pull request merged**